plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerConfig.java

```diff
@@ -70,6 +71,24 @@ public FakerConfig setLocale(String value)
         return this;
     }
 
+    @Max(2)
+    @Min(0)
+    public double getSequenceMinDistinctValuesRatio()
+    {
+        return sequenceMinDistinctValuesRatio;
+    }
+
+    @Config("faker.sequence-min-distinct-values-ratio")
```

Comment:
Can we avoid exposing this as a config ? Just define it as a constant in the code. Its not easy to grasp what this is and I don't see a reason why users need to tune it